low-level/files/opt/seagate/sspl/setup/sspl_setup.py

```diff
@@ -204,8 +203,9 @@ def validate(self):
         pass
 
     def process(self):
-        # TODO: Import relevant python script here for further execution.
-        pass
+        from cortx.sspl.lowlevel.files.opt.seagate.sspl.setup import sspl_config
```

Comment:
Please keep this import statement at top.

Reply:
Import structure is actually as mentioned by Ujjwal. since these import statements are for each setup stage, I think he has included it in this fashion